src/python/run bf testes/bf/1.g: OK (24ms)
src/python/run bf testes/bf/2.g: Fail
  ** STDOUT **
    s 0
  ** STDERR **
    Traceback (most recent call last):
      File "main.py", line 18, in <module>
        main()
      File "main.py", line 13, in main
        bf.execute(vertexes, edges, vertexes[0])
      File "/home/rodrigo/graphun/src/python/bf.py", line 15, in execute
        printBf(V, s)
      File "/home/rodrigo/graphun/src/python/bf.py", line 31, in printBf
        printPath(s, v)
      File "/home/rodrigo/graphun/src/python/bf.py", line 38, in printPath
        elif pi[v] == None:
    KeyError: 'r'
  ** ERRORS **
    caminho n達o encontrado: s y
    caminho n達o encontrado: s t
    caminho n達o encontrado: s z
    caminho n達o encontrado: s x
src/python/run bf testes/bf/3.g: OK (26ms)
src/python/run bf testes/bf/4.g: OK (30ms)
src/python/run bf testes/bf/5.g: OK (35ms)
src/python/run bf testes/bf/6.g: OK (1s 100ms)
src/python/run bf testes/bf/7.g: OK (8s 823ms)
src/python/run bf testes/bf/8.g: OK (30s 138ms)
src/python/run bf testes/bf/9.g: 